### PR TITLE
fix description & time to read

### PR DESCRIPTION
**Pull request by eugene-matvejev**

### CODE REVIEW

File: src/components/card.jsx [hunk 1]
```diff
@@ -16,33 +16,33 @@ const Card = ({
...
+                <p className={css.card__details}>
+                    {publishDate} | {author.name} | {description.childMarkdownRemark.timeToRead} min read
+                </p>
+                <p
+                    data-cy="card-description"
+                    className={css.card__description}
+                    dangerouslySetInnerHTML={{ __html: description.childMarkdownRemark.html }}
+                >
+                    {
+                        // description.childMarkdownRemark.excerptAst && render(description.childMarkdownRemark.excerptAst)
```
Comment (cchetcuti) [inline src/components/card.jsx]: Why is this commented out? This should use Excerpt as it gives us the '...' in the end and we can limit the number of characters.

Comment (cchetcuti) [reply src/components/card.jsx]: Number of characters : 160

**Comment by eugene-matvejev** [comment]
lets merge this
and i keep work on it

**Comment by andgreendigital** [comment]
> lets merge this
> and i keep work on it

Unfinished code shouldn't be merged! There is no reason we can't keep working on this _before_ we merge it.

**Comment by cchetcuti** [comment]
> lets merge this
> and i keep work on it

@eugene-matvejev Description still needs to be fixed (not just colour and title) as it will not pass the ACs. Let's keep this till we have all changes :)

**Comment by eugene-matvejev** [comment]
AC?

**Comment by eugene-matvejev** [comment]
@andgreendigital it fixes a bugs, which are blocking stuff, if there are more bugs, i would better fix in separate PRs, so we can move on at least with this bit

**Comment by eugene-matvejev** [comment]
it is safe to merge, it will fail till new Card containers aren't merged

**Pull request merged**